nucypher/cli/commands/ursula.py

```diff
@@ -350,15 +358,24 @@ def init(general_config, config_options, force, config_root, key_material):
         return click.get_current_context().exit(1)
 
     click.clear()
-    emitter.echo(
-        "Hello Operator, welcome on board :-) \n\n"
-        "NOTE: Initializing a new Ursula node configuration is a one-time operation\n"
-        "for the lifetime of your node.  This is a two-step process:\n\n"
-        "1. Creating a password to encrypt your operator keys\n"
-        "2. Securing a taco node seed phase\n\n"
-        "Please follow the prompts.",
-        color="cyan",
-    )
+    if with_mnemonic:
+        emitter.echo(
+            "Hello Operator, welcome back :-) \n\n"
```

Comment:
Is it possible for the init to be called with `--with-mnemonic` the first time ever or is there something preventing that? In which case it may not be "welcome back" necessarily.

Reply:
I'm working under the assumption that they are in a possession of a mnemonic that they received on first `init`.  I suppose it's possible for someone to generate their own mnemonic and bring it along.  What do you think?

Reply:
Yeah. I'm wondering if `--with-mnemonic` my be used similar to how `--key-material` would be if you have your own source of randomness.

Reply:
✔️